lib/workers/repository/update/branch/auto-replace.ts

```diff
@@ -196,6 +196,28 @@ async function checkExistingBranch(
   return existingContent;
 }
 
+/**
+ * Check if an update from `current` to `newString` should be performed and return 1 if so.
+ *
+ * @remarks
+ * Useful for counting the number of updates to do.
+ *
+ * @param {string|undefined} current The current value (if undefined then no update is required)
+ * @param {string|undefined} newString The new value (if undefined then no update is required)
```

Comment:
```suggestion
 * @param current The current value (if undefined then no update is required)
 * @param newString The new value (if undefined then no update is required)
```
types not required for tsdoc only for jsdoc style 🤗